Core/src/main/java/com/plotsquared/core/events/post/PostPlotClearEvent.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * PlotSquared, a land and world management plugin for Minecraft.
+ * Copyright (C) IntellectualSites <https://intellectualsites.com>
+ * Copyright (C) IntellectualSites team and contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+package com.plotsquared.core.events.post;
+
+import com.plotsquared.core.events.CancellablePlotEvent;
+import com.plotsquared.core.events.PlotEvent;
+import com.plotsquared.core.events.Result;
+import com.plotsquared.core.plot.Plot;
+import com.plotsquared.core.plot.PlotId;
+
+/**
+ * Called after a {@link Plot} was cleared.
+ *
+ * @since 7.2.1
```

Comment:
This should be `TODO` as we have a task that will go through and replace them automagically